src/main/java/org/mapdb/Serializer.java

```diff
@@ -25,108 +24,124 @@
 import java.util.*;
 
 /**
- * Provides serialization and deserialization
+ * This interface specifies how Java Objects are serialized and de-serialized
+ * and also how objects are compared, hashed and tested for equality for use
+ * with MapDB.
+ * <p>
```

Comment:
It is a way of separating comments so it looks nicer in the IDEs. The paragraph that follows is not related to the overall description of the interface and so, it is separated.